### ISSUE TITLE
Feed doesn't load sometimes

### ISSUE THREAD

**Issue by michaelhall923** [stemstr/Client#89]
Sometimes the page will load and nothing will show up on the feed

**Comment by michaelhall923** [comment]
NDK has a planned feature called buffered subscriptions which should help with this. It will consolidate similar subscriptions into one

**Comment by SamSamskies** [comment]
The reactions are also causing a ton of subscriptions.

**Comment by SamSamskies** [comment]
I don't think this is an issue anymore. Seems to have be resolved with the note refactoring.

**Comment by michaelhall923** [comment]
Found one case where it's still broken, added to OP
